python/pyspark/sql/connect/client/artifact.py

```diff
@@ -187,6 +188,19 @@ def _parse_artifacts(self, path_or_uri: str, pyfile: bool, archive: bool) -> Lis
             return [artifact]
         raise RuntimeError(f"Unsupported scheme: {parsed.scheme}")
 
+    def _parse_forward_to_FS_artifacts(self, local_path: str, dest_path: str) -> List[Artifact]:
```

Comment:
BTW, this one too. Shouldn't we do it together with `_parse_artifacts`?

Reply:
I don't think so. The arguments are different. For `_parse_forward_to_FS_artifacts`, we have `local_path` and `dest_path` and the 2 paths are different.